src/services/line-stat-getter.js

```diff
@@ -1,109 +1,136 @@
-'use strict';
-var _ = require('lodash');
-var P = require('bluebird');
-var moment = require('moment');
-var BaseStatGetter = require('./base-stat-getter');
-var Database = require('../utils/database');
-var Interface = require('forest-express');
+import { find, sortBy, map, values as _values } from 'lodash';
+import { map as _map } from 'bluebird';
```

Comment:
[Code style] This is so misleading:
- `_values` stands for `_.values`
- `_map` stands for .... `P.map` (whereas `map` stands for `_.map`)